hack/move-api-packages.sh

```diff
@@ -0,0 +1,135 @@
+#!/bin/bash
```

Comment:
> Are you expecting this to stick around?

Not in the sense of "this script needs to be pretty".  I want the script in this pull so that people can see if it does roughly what they expect instead of reviewing the 20k diff.